templates/core/terraform/resource_processor/vmss_porter/cloud-config.yaml

```diff
@@ -59,18 +59,23 @@ write_files:
       set -o errexit
       used_percent=$(df / --output=pcent | tail -1 | sed 's/[^0-9]//g')
       echo "Used disk space percent: $${used_percent}"
-      if (( used_percent > 70 )); then
+      if (( used_percent > 75 )); then
         echo "Free space too low, pruning..."
         docker system prune -f
       fi
     permissions: '0755'
 
 runcmd:
+  # Those are useful live debug commands. Check the docs for details
```

Comment:
Perhaps add link to doc?

Reply:
Added a TODO as I'll handle all docs in a separate PR.